sdk/search/Azure.Search.Documents/src/autorest.md

```diff
@@ -58,6 +58,13 @@ directive:
     $["discriminator"] = "@odata.type";
 ```
 
+```yaml
```

Comment:
Nit: Like other transforms, can we add a description here?

Reply:
Done!